terraform/configs.tf

```diff
@@ -2,13 +2,13 @@
 # See LICENSE file for licensing details.
 
 module "k8s_config" {
-  source = "git::https://github.com/canonical/k8s-bundles//terraform/manifest?ref=main"
+  source = "./manifest/"
```

Comment:
if we change this here, do remote deployments still pull in this module?

I assume this was mostly tested when the repo was already checked out locally

Reply:
Using GH actions, i've confirmed that this change is functional.